nixos/modules/services/continuous-integration/hail.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  cfg = config.services.hail;
+in {
+
+
+  ###### interface
+
+  options.services.hail = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = "Enable Hail Auto Update Service.";
```

Comment:
Can you provide a better description.  I think this is likely to be one of the most useful option, but the current description does not help at discovering this service, nor how it can be used to automatically poll for updates.